protocol/src/protocol_generated_verifier.rs

```diff
@@ -52,19 +52,14 @@ pub mod ckb {
 
                 let vtab = tab.vtable();
                 let vtab_num_bytes = vtab.num_bytes();
-                let object_inline_num_bytes = vtab.object_inline_num_bytes();
```

Comment:
This is generated by an old version of cfbc, please install 0.19 and regenerate it